torch/onnx/utils.py

```diff
@@ -412,6 +411,9 @@ def _model_to_graph(model, args, verbose=False,
     param_names = input_and_param_names[len(input_and_param_names) - len(params):]
     params_dict = dict(zip(param_names, params))
 
+    if training is None or training == TrainingMode.EVAL:
+        params_dict = torch._C._jit_pass_onnx_initializer_based_peephole(graph, params_dict)
```

Comment:
Looks like this is going to be place where optimizations happen for eval mode. Maybe rename to `_jit_pass_onnx_eval_peephole`.

Can we even refactor the code to put this pass into `_optimize_graph`? There was duplication of creating `params_dict` before and after `_optimize_graph` call. Let's take this opportunity for some clean up.

Reply:
moving the pass under _optimize_graph produces following exception in ort:
File "/home/ksenija/anaconda3/envs/pytorch/lib/python3.7/site-packages/onnxruntime/capi/session.py", line 107, in run
    raise ValueError("Model requires {} inputs. Input Feed contains {}".format(num_required_inputs, num_inputs))
ValueError: Model requires 3 inputs. Input Feed contains 1